tools/tune/src/src_tiny_int16.m

```diff
@@ -27,14 +27,15 @@
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 %
 
-fs1 = [8e3 16e3 32e3 44.1e3 48e3 50e3];
-fs2 = [8e3 16e3 32e3 44.1e3 48e3 50e3];
-fsm = [0 0 0 0 1 0; ...
-       0 0 0 0 1 0; ...
-       0 0 0 0 1 0; ...
-       0 0 0 0 1 0; ...
-       1 1 1 1 0 1; ...
-       0 0 0 0 1 0; ...
+fs1 = [8e3 16e3 32e3 44.1e3 48e3 50e3 96e3];
+fs2 = [8e3 16e3 32e3 44.1e3 48e3 50e3 96e3];
+fsm = [0 0 0 0 1 0 0; ...
+       0 0 0 0 1 0 0; ...
+       0 0 0 0 1 0 0; ...
+       0 0 0 0 1 0 0; ...
+       1 1 1 1 0 1 0; ...
```

Comment:
Actually, now, I realize that on the 5th row it should be: `{1 1 1 1 0 1 1}` instead of `{1 1 1 1 0 1 0}` because we want to convert 96 to 48. I'll update this.

Reply:
> Actually, now, I realize that on the 5th row it should be: `{1 1 1 1 0 1 1}` instead of `{1 1 1 1 0 1 0}` because we want to convert 96 to 48. I'll update this.

Done